action.yml

```diff
@@ -6,6 +6,9 @@ inputs:
   slack-webhook-url:
     description: 'Slack channel webhook URL to report to'
     required: true
+  parent-job-status:
```

Comment:
I think we can call this input `job-status` since this actions does not have a `jobs` field

Reply:
hmm, I feel that job-status is the current one, the one which is being executed, but the parent-hob-status is the job status which called this job

Reply:
There is only one job.
This is an action.